### PR TITLE
[main] Update flake.lock + generated files

### PR DESCRIPTION
**Pull request by GaetanLepage**

### CODE REVIEW

File: generated/efmls-configs.nix [hunk 1]
```diff
@@ -49,7 +49,9 @@
   blade = {
     formatter = {
       lang = "blade";
-      possible = [ "blade_formatter" ];
+      possible = [
+        "blade_formatter"
+      ];
```
Comment (MattSturgeon) [inline generated/efmls-configs.nix:54]: Seems like `nix fmt` failed? The generate-files script is _supposed_ to format the generated files automatically...

**Comment by MattSturgeon** [comment]
Only generated/none-ls.nix was actually changed, the rest seem to be formatting errors

**Review by MattSturgeon — APPROVED** [review]

**Comment by MattSturgeon** [comment]
This still looks good, but it'd be nice to have https://github.com/nix-community/nixvim/pull/1956 merged first to verify this with the new test. We'd also be about to double check the update CI is correctly fixed.

**Comment by MattSturgeon** [comment]
Update, the CI works correctly (https://github.com/nix-community/nixvim/actions/runs/10179460015), and created https://github.com/nix-community/nixvim/pull/1958 which [failed](https://buildbot.nix-community.org/#/builders/2655/builds/3) the new test (as expected).

```
> none-ls: The following are not declared in plugins/none-ls/packages.nix:
> - terragrunt_validate
> - terragrunt_fmt
```

---

I did notice an issue with the commit message though, it generated as:
```
generated: Updated none-ls.nix
- Updated rust-analyzer.nix
```
But it should be:
```
generated: Update

- Updated none-ls.nix
- Updated rust-analyzer.nix
```

So it seems this isn't working correctly: https://github.com/nix-community/nixvim/blob/3789c696584406fdfd5e9bfcc13d41c45ae5715c/flake-modules/updates/default.nix#L59-L64

**Comment by MattSturgeon** [comment]
Darwin specific issue? https://buildbot.nix-community.org/#/builders/845/builds/39/steps/1/logs/stdio

**Comment by GaetanLepage** [comment]
@Mergifyio queue

**Pull request merged**